doc/config/pkcs11

```diff
@@ -0,0 +1,35 @@
+# Include common configuration options.
+@INCLUDE_PATH = doc/config
+@INCLUDE = common
+
+# Basic project information.
+PROJECT_NAME = "PKCS11"
+PROJECT_BRIEF = "PKCS11 Cryptoki Library"
+
+# Library documentation output directory.
+HTML_OUTPUT = pkcs11
+
+# Generate Doxygen tag file for this library.
+GENERATE_TAGFILE = doc/output/pkcs11.tag
+ENABLE_PREPROCESSING = YES
+MACRO_EXPANSION = YES
+PREDEFINED="CK_DECLARE_FUNCTION(return, name)=return name"
+
+# Directories containing library source code.
+INPUT = doc \
+        doc/lib \
+        libraries/3rdparty/pkcs11 \
+        libraries/abstractions/pkcs11 \
+        libraries/freertos_plus/standard/pkcs11/include \
+        libraries/freertos_plus/standard/pkcs11/src \
+        libraries/abstractions/pkcs11/mbedtls \
+        libraries/abstractions/pkcs11/include \
+        libraries/freertos_plus/standard/utils/include/ \
+        libraries/freertos_plus/standard/utils/src/ \
+        vendors/pc/boards/windows/ports/pkcs11 
```

Comment:
Windows port is used here, this file will vary between ports but should be okay to use (and has historically) for the purposes of generating the dox